Realm/RLMObjectBase.h

```diff
@@ -33,6 +33,9 @@ RLM_ASSUME_NONNULL_BEGIN
 
 + (NSString *)className;
 
+/// Returns whether the class is included in the default set of classes persisted in a Realm.
```

Comment:
@jpsim this is more for component makers, so if they use realm as an implementation detail, the end user won't have to specify class subsets.